.github/workflows/run-bdd-tests.yml

```diff
@@ -20,5 +20,11 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: Build BDD covered packages and dependencies
         run: yarn workspace @ovh-ux/manager-container-app start
-      - name: Run tests
+      - name: Discover packages with Jest tests
+        id: discover-packages
+        run: |
+          node -e "const { execSync } = require('child_process'); const packagesWithTests = execSync('find packages -name \"jest.config.js\" -exec dirname {} +').toString().split('\n').filter(Boolean); console.log(`PACKAGES_WITH_TESTS=\${packagesWithTests.join(' ')}`);"
+        shell: bash
```

Comment:
**question:** What's exactly the purpose of this step?

Not sure that listing all locations where a `jest.config.js` exist is adding
any values.

And you shouldn't rely on node to simply perform a `find` command.

Reply:
Also, it looks like there's an bad substitution. See the logs here:
https://github.com/ovh/manager/actions/runs/7399950659/job/20132508846?pr=10618